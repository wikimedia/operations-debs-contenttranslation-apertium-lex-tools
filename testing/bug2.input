^пограничен<adj><mfn><pl><nom><ind>/border<adj><mfn><pl><nom><ind>$ ^област<n><f><pl><nom><ind>/field<n><pl><nom><ind>/area<n><pl><nom><ind>/province<n><pl><nom><ind>/district<n><pl><nom><ind>$^.<sent>/.<sent>$
