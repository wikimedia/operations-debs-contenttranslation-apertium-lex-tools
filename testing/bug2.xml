<rules>
<rule comment="103"><match lemma="област"><select lemma="area"/></match></rule>
<rule comment="44"><match lemma="граничен"><select lemma="border"/></match></rule>
<rule comment="32"><match lemma="пограничен"><select lemma="border"/></match></rule>
</rules>
