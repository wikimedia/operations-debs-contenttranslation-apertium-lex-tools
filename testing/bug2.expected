^пограничен<adj><mfn><pl><nom><ind>/border<adj><mfn><pl><nom><ind>$ ^област<n><f><pl><nom><ind>/area<n><pl><nom><ind>$^.<sent>/.<sent>$
